app/assets/javascripts/blacklight/core.js

```diff
@@ -14,7 +14,8 @@ Blacklight = function() {
 }();
 
 // turbolinks triggers page:load events on page transition
-// If app isn't using turbolinks, this event will never be triggered, no prob. 
+// If app isn't using turbolinks, this event will never be triggered, no prob.
+// Support for Turbolinks classic
 $(document).on('page:load', function() {
```

Comment:
Maybe.. I'm trying to figure out what the best way to support this forward and backwards compatible is. I'm running into issues where activate will fire twice (once from `ready` and once from `turbolinks:load`)